spdmlib/src/protocol/algo.rs

```diff
@@ -49,47 +49,10 @@ pub const SPDM_RANDOM_SIZE: usize = 32;
 pub const SPDM_MAX_HASH_SIZE: usize = 64;
 pub const SPDM_MAX_ASYM_KEY_SIZE: usize = 512;
 pub const SPDM_MAX_DHE_KEY_SIZE: usize = SECP_384_R1_KEY_SIZE;
+pub const SPDM_MAX_PSK_SIZE: usize = 32;
```

Comment:
> why we need limit this? The size of PSK should be agnostic to spdmlib, because only secrete lib knows PSK.

Oh, this is added by mistake, already removed in latest commit.